### ISSUE TITLE
Various Compatibility questions/issues.

### ISSUE THREAD

**Issue by jfrazierjr** [mouse0270/fantasy-rpg-ui#21]
There are a few incompatibilities with Minimal UI, as well as various mods which add stuff to the header of a character sheet.  

Example: 

In minimal UI, there is an option for uncluttering the left nav bar so you can only have the buttons show.

![image](https://user-images.githubusercontent.com/2130984/199148843-3ff626c0-f9e3-4235-a767-ffbf81520063.png)

For example, the "tool" menu (left most) will only show the currently selected tool), and then the sub tool(the right most) will show a common selection of the subtools. to keep the UI exceedingly uncluttered.  

As well, Minimal UI has support for the macro bar positioning, but that does not work with your Module loaded.

I suspect this is due to the load order of the css and scripts in that yours runs after Minimal UI.   

Finally, it took me a few minutes to figure out the side tabs on a player character and monster sheet(the rings to the right) were like tabs in a book.    I don't know if this is default or not, but for me with all my mods, the "tabs" on the right side don't have a background, border, or text:

![image](https://user-images.githubusercontent.com/2130984/199149935-5237b598-3ed6-4f8e-9b4d-8220e83329e3.png)

I don't know if you would consider adding support for Minimal UI's features, but if not you might want to put that into the partial compatibility list as a number of it's items clash with your's.    For some of this, I can fairly easily use the Custom CSS module(using it anyway for my mobile phone guy).  

Combat Carousel:   The default styling on the combat carousel is lost.   They normally have rounded clip and the current init token is slightly larger as well.   Basically, it loses its default styling and I don't know if the change is your styling or simply lost default styling.  
 
Monk's Scene Navigation: seems to work.. no styling on the navigation items, so you might want to put that in with the no Theme checkbox in Partial support

FYI, I LOVE the sheets that show the header, as well as close as a tab to the side.. that is an incredible styling touch that really makes your UI stand out.  Thanks for the work you have put in and I look forward to any other fixes/additions you plan to add to it!

Things that work:
Monks Hotbar Expansion:   Honestly, this works amazingly well..   
Always HP:  This looks amazing
Monk's Scene Navigation: seems to work.. no styling on the navigation items, so you might want to put that in with the no Theme checkbox in Partial support
Monk's Token Bar: unless you count the actual token bar theming(not sure if that's a big deal or not), this works very well, the request rolls dialog looks really nice

 *Developer Edit*
- [ ] Minimal UI Fixes
- [ ] Combat Carousel Fixes
- [x] Monks Scene Navigation - Add Support For Theming
- [ ] Monks Tokenbar - Add Support for Theming

**Comment by mouse0270** [comment]
I don't use Minimal HUD as I wrote my own minimal HUD called TidyHUD in libThemer. I can install it and take a look at how easy or hard it will be to add compatibility.

There is a setting in the Configure Theme to make the Header buttons either always active or show on hover. Could you better explain what you mean by 
> the "tabs" on the right side don't have a background, border, or text
I think you mean they are invisible until hover, but that is intended to clean up the interface, however, it should be an option you can toggle in the Configure Theme. Please expand on what you think this should do as I might be a misunderstanding.

Combat Carousel is another module I do not use, so I can install it and try to fix the issues with it, just give me a day or so.

Monk's Scene Navigation: Depending on how complex this css is, I will either add theme support for it, or I will just say it doesn't break the UI. lol

Monks Token Bar is something I want to add theme support to, currently I am pretty sure it works by default, but I will double check.

**Comment by jfrazierjr** [comment]
I will download your TinyHUD later tonight and see if it meets my needs(it likely will) as I have also found a fairly big issue (for me anyway) in MinimalUI.   Namely, the AV panel **when left/right docked** has it's control buttons covered by the Player List and are unusable.   I semi quickly get around by adding custom CSS to hide the player Panel, but that's not a very elegant solution.   Unfortunately, I have not had time to report this to the MinimalUI dev and hope to get that case opened tonight.

As for the tabs, yes, invisible until on hover is what I meant.   Mainly this is a bit of confusion vs your documentation:
https://user-images.githubusercontent.com/564874/193952294-5032d9e6-9d5a-4174-a27e-984c4ae3b1e0.png

As above, the right side tabs for "Show Players", "Sheet", etc are always visible, so you might want to note that in your ReadMe if they are now intended to be on hover only, and likely show a quick example of it's use.

**Combat Carousel**: not a huge deal, but would be nice to have something in between your style and theirs.    IIRC, that module allows for various size and it's own mini themes with quite a lot of options.  It MIGHT be easier for you to just make your own or just mark it as partially compatible due to the number of use cases that would need to be tested.    


**Monk's Scene Navigation**:   Yea, not an issue honestly and is super low priority TBH.   more of a nice to have at the bottom of the stack.

**Monks Token Bar** ditto the scene nav mod, nice to have but bottom of the stack.

Again, thanks a ton for all the work you have done it's amazing.

**Comment by mouse0270** [comment]
You already have TindyHUD, its packaged with libThemer which is a requirement of this Theme. Tidy HUD does very limtited things, really all it does is give you control over the size and position of the elements on the UI. But I am always happy to add features to it, you can request them over at https://github.com/mouse0270/lib-themer

The preview image has the option to hide header tabs turned off to give users an idea of what they look like. You should be able to go to the Sidebar > Configure Theme > Fantasy RPG UI > and then go through the options and toggle that setting off and they will always be displayed. I personally prefer them hidden as they aren't something I need to always see.
*Personal Note:* I should make it so you can toggle this setting globally and not have to toggle it for each type of theme-able option

Combat Carousel, I installed the module and will check it out and see if I can easily theme it.

Monks Modules: I always planned to theme Tokenbar because I use it, as for Enhanced Scene navigation, from a quick look it should be easy enough to theme, but I don't use this module so if I do theme it and miss anything just let me know.

**Comment by mouse0270** [comment]
Added Support for Monks Hotbar Expansion and Scene Browser as of v0.4.0 https://github.com/mouse0270/fantasy-rpg-ui/releases/tag/0.4.0

I still have to take a more in depth look at Minimal UI, Combat Carousel and Monks Tokenbar

**Comment by jfrazierjr** [comment]
Ok, so after doing some more digging, I realized that Combat Carousel actually had quite small bit of theming.     The theming I as seeing and liked so much for Combat Carousel actually came from **DnD5e UI ON TOP of Combat Carousel**  

Here is default Foundry with CC running:

![image](https://user-images.githubusercontent.com/2130984/199780781-cc8be348-67f5-4989-bcf8-dcec7074cd21.png)

Note that the current combatant is "highlighted" but barely.   Also note that CC has a number of options for the size of each actor in the Carousel and I have mine set to extra small.

Adding on Fantasy RPG UI really makes no changes to the CC at all currently.  

Now, Here is CC with DnD5e UI installed which really punches up the presentation of CC quite a lot:
![image](https://user-images.githubusercontent.com/2130984/199782225-bc65d6cd-7ecf-4ba5-8bdc-43e9cd8ec072.png)

Now of course I am not asking you to copy someone else's work, but perhaps this can serve as a bit of inspiration for your themes via lib themer and fantasy UI. 

I am in NO way great at css, but I did a quick play around and got something a bit closing using the custom CSS module that makes the default a bit more like fantasy video game theme but am at the limits of my personal css abilities:

again, for inspiration purposes:

![image](https://user-images.githubusercontent.com/2130984/199783136-4b390493-b066-47a8-8572-71483c28420f.png)

I had tried to make the current init token bigger but that messed stuff up, so it's likely in a js modifying settings rather than pure CSS(or again beyond my personal abilities!)   This works for me for now.

**Comment by jfrazierjr** [comment]
One more thing, i just noticed.. For Monk's Scene Navigation you might want to either mark that as Partially Compatible.  

Yours with Monk's Scene Nav turned off

![image](https://user-images.githubusercontent.com/2130984/199784510-cc348be8-087d-488e-89f3-6729ed6fb372.png)

With FRPGUI/Monk Scene Nav and "Display Background" Turned ON it looks REALLY good:

![image](https://user-images.githubusercontent.com/2130984/199786155-f70d51f1-bd1b-4970-bbbb-c6ea9f922f11.png)



With FRPGUI + Monk's Scene Nav turned with "Display Background" turned OFF it goes back to the default. 


![image](https://user-images.githubusercontent.com/2130984/199785340-2c1fdd88-7289-4859-bf8a-8bfff5a1f54b.png)

Basically, for any screenshots of the Nav you might make for your module, you likely want to call out that to get the nice themed texture for each Nav Item, they likely also need to turn on the Display Background check box in Monk's as well.
